server/src/main/java/org/opensearch/repositories/Repository.java

```diff
@@ -237,6 +239,14 @@ default RepositoryStats stats() {
      */
     boolean isReadOnly();
 
+    /**
```

Comment:
How would a callers use this field?  It seems like it makes handling the `system'ness` of a responsible of the callers, rather than the responsibility of the implementer.